db/migrations/20180102183100_add_stack_to_buildpack_table.rb

```diff
@@ -0,0 +1,30 @@
+require 'yaml'
+
+def default_stack
```

Comment:
Since there is some logic here, it would be nice to write a migration test. 

See examples here: https://github.com/cloudfoundry/cloud_controller_ng/tree/master/spec/migrations

Reply:
:+1: Will do.